modules/pam_faillock/main.c

```diff
@@ -111,10 +129,11 @@ do_user(struct options *opts, const char *user)
 	int rv;
 	struct tally_data tallies;
 	struct passwd *pwd;
+	const char *dir = (opts->dir != NULL) ? opts->dir : FAILLOCK_DEFAULT_TALLYDIR;
```

Comment:
Please replace this repeated expression with a function `const char *get_tally_dir(struct option *opts)` implemented in faillock_conf.c.

Reply:
Done